### ISSUE TITLE
WEBGL : make orbitControl() smoother and more continuous

### ISSUE THREAD

**Issue by a-y-u-s-h** [processing/p5.js#2497]
#### Nature of issue?

- [ ] Found a bug
- [x] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [ ] Core
- [ ] Data
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [x] WebGL
- [ ] Other (specify if possible)

#### Feature enhancement details:

I think `orbitControl()` function in WEBGL mode should behave more like (or exactly like..) [this](https://threejs.org/examples/?q=text#webgl_buffergeometry_instancing2) sketch. As of right now, when user releases button after click, the camera position goes back to previous position instantly, which doesn't feel like desired behaviour.

**Comment by diwi** [comment]
Here is a little fiddle to test. The used library is based on the PeasyCam.

https://jsfiddle.net/t2c8aa1h/

LMB: orbit
MMB: pan
RMB: zoom
dlbclick: reset

**Comment by lmccart** [comment]
i like this suggestion, @mlarghydracept what do you think?

**Comment by mlarghydracept** [comment]
Yeah I think it is probably the more expected behavior.

**Comment by mcanepa** [comment]
I will also suggest to have the possibility to invert the control movement.

To better understand what I'm saying, see this [demo](http://showroom.littleworkshop.fr/). In that example, you can move the camera in a more natural way around the room.

Maybe something like `orbitControl(INVERT)`

**Comment by AidanNelson** [comment]
I've started working on this with [this](https://threejs.org/examples/?q=control#misc_controls_orbit) as a reference (three.js' 'orbitControls' rather than the 'trackballControls' linked above). The main difference between the two according to [this stackexchange post](https://stackoverflow.com/questions/18581225/orbitcontrol-or-trackballcontrol) is that orbitControl maintains the same cameraUp vector, which helps the user not lose their orientation by accident. 

@mcanepa I added x/y sensitivity parameters to orbitControl with #2975, which allows inversion (by entering a negative number).  The current (0.6.1) default mode is to have the Y-Axis inverted (dragging mouse down makes camera look up) whereas the new default is non-inverted Y-axis. I just need to figure out why it isn't passing its TravisCI check...
